src/components/bind-results/bindResults.html

```diff
@@ -0,0 +1,43 @@
+<div>
+  <div ng-if="!ctrl.error">
+    <div ng-if="!(ctrl.binding | isBindingReady)" ng-class="{'text-center': !ctrl.progressInline}">
+      <h3 class="bind-pending">
+        <span class="fa fa-spinner fa-pulse fa-fw" ng-class="{'fa-3x': !ctrl.progressInline}" aria-hidden="true"></span>
```

Comment:
I've used what was already there for both bind and adding to project. When not inline, I'm not sure the patternfly large spinner is a large as we want here. Thoughts?